specification/gedcom-03-datamodel.md

```diff
@@ -1221,6 +1221,30 @@ An event structure asserts the event did occur if any of the following are true:
     ````
     
     </div>
+    
+    <div class="note">
+    
+    That a `DATE` indicates an event occurred was introduced to in version 5.4 (1995), and is generally followed by applications.
```

Comment:
I can't parse this sentence.  Not sure what "to in" means, and also reads awkward starting the sentence with "That".